### DISCUSSION TITLE
Skiovox FIX for ChromeOS version 120 (Newest version btw)

### DISCUSSION THREAD

**Discussion by v4pl** [3kh0/ext-remover#929] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Made fully by @Fighterhen63, I do not hold any respect for this..
Here are the steps..

If you find that u don't have sign in as existing user button when u are on step 7, do this: When you are at the normal sign in screen, turn off wifi, see if u have sign in as existing user there, if u do, click it, then continue to do the method:

Sign out of your chromebook.

Turn off your wifi

Click one of the apps

As you click one of the apps, press alt+shift+s as fast as possible

This should open the tool bar, wait until the No wifi screen show up.

Click the screen brightness settings in the tool bar and click settings(the cog) nothing should happen(tool bar will be closed) and thats normal.

Open chrome vox(ctrl+alt+z) then click resources (Like open the menu which has the links like the Search TO method from the old methods), then click on one of the links. Now close chrome vox (same keybinds).

Click sign in as existing user put in your school account code and password. (Normal school login.)
If you did it successfully, you should be signed in to school account in the kiosk app and you should see something saying "muti user sign in disabled on this chromebook" just press the ESC button.

You should see your school account browser there just close it, you should also see a setting menu somewhere, just try to turn on wifi then close that window. The last window is the fully unblocked one. You can also open incognito, DO NOT because it is still monitored to your school account so don't use that. Something you need to do it click the three dots and open "New Window" this will allow windows to be opened so you can install skiviox. This should be your primary window.

If anyone has any questions feel free to reach out to Fighterhen63!

**Comment by kiddmarco125** [comment]
Yeah I don't have sign in as existing user when I turn off wifi

**Reply by v4pl** [reply]
Um screen shot and show me

**Reply by v4pl** [reply]
I will try to help u

**Reply by kiddmarco125** [reply]
![IMG_20240215_102947.jpg](https://github.com/3kh0/ext-remover/assets/158217466/dc34fe07-bd82-4590-8282-5886e7b7ebe9)

**Reply by v4pl** [reply]
um did you sign in to this chromebook before?

**Reply by v4pl** [reply]
causes for me i have to sign into the chromebook at least once and then it should work

**Reply by kiddmarco125** [reply]
I'll try but I don't think it'll work

**Reply by kiddmarco125** [reply]
Yeah I did just that and it still doesn't show sign in as existing user

**Reply by Kalbrooooo** [reply]
I don’t have a tab after I close settings, why?

**Reply by Kalbrooooo** [reply]
Kiddmarco just use a different app and it should work

**Reply by kiddmarco125** [reply]
I tried it on every app and nothing

**Reply by v4pl** [reply]
Yo @kiddmarco125 press alt shift ctrl but don't power wash then try again

**Comment by ghost** [comment]
Im not a 2A user, will it still work?

**Reply by v4pl** [reply]
Ya think so

**Comment by lolpc100** [comment]
actually
use with sign in with another user

**Comment by lolpc100** [comment]
log in
and press sign in with another user

**Comment by lolpc100** [comment]
> Made fully by @Fighterhen63, I do not hold any respect for this.. Here are the steps..
> 
> If you find that u don't have sign in as existing user button when u are on step 7, do this: When you are at the normal sign in screen, turn off wifi, see if u have sign in as existing user there, if u do, click it, then continue to do the method:
> 
> Sign out of your chromebook.
> 
> Turn off your wifi
> 
> Click one of the apps
> 
> As you click one of the apps, press alt+shift+s as fast as possible
> 
> This should open the tool bar, wait until the No wifi screen show up.
> 
> Click the screen brightness settings in the tool bar and click settings(the cog) nothing should happen(tool bar will be closed) and thats normal.
> 
> Open chrome vox(ctrl+alt+z) then click resources (Like open the menu which has the links like the Search TO method from the old methods), then click on one of the links. Now close chrome vox (same keybinds).
> 
> Click sign in as existing user put in your school account code and password. (Normal school login.) If you did it successfully, you should be signed in to school account in the kiosk app and you should see something saying "muti user sign in disabled on this chromebook" just press the ESC button.
> 
> You should see your school account browser there just close it, you should also see a setting menu somewhere, just try to turn on wifi then close that window. The last window is the fully unblocked one. You can also open incognito, DO NOT because it is still monitored to your school account so don't use that. Something you need to do it click the three dots and open "New Window" this will allow windows to be opened so you can install skiviox. This should be your primary window.
> 
> If anyone has any questions feel free to reach out to Fighterhen63!

actually use sign in as a existing user

**Reply by Gaboom63** [reply]
@lolpc100 I am pretty sure this is patched as well due to the esc button being patched. The latest version at least, base 120 is fine.  On the latest version of Chrome alt shift s is patched as well if I'm not mistaken as well as the esc button. I will look into a new method after school.  I will find a new way trust lol.

**Reply by v4pl** [reply]
you better cause i'm stuck at 121 if you do I will pay a decent about of money for it

**Reply by CGRmags** [reply]
@vaperbest u should give me mine first pls

**Comment by Gaboom63** [comment]
Lol I don't need money, I do it for the people.

On Tue, Mar 12, 2024, 10:10 AM
V̶̛̼̰̰͓̲̞̤̝͕̝̱̯̠̲̍̉͗̾͗̂̒̃̈́͗̎͂͋̍̑͌́͋͝͝Ŝ̵̥̘̜̬̲͚͖̤̥̠̬̻̲͚̟̮̂͐͋̇͊̔́͋̋̾̃́̂́̾́̅̈́̍̑̈́͂̚͘͝ͅḺ̴̢̜̩͎̩͖͖̤̣̦̗̮͒͛̾̈́̐͋̍̇͑͐͂̄̄̊́͋͘̚͠A̴̡̢̛̻̪̙̝̙̤̫̞̳̥̱͉̫̰̤̹̤̟̯͖̥̠̳̥̍̊̈̓͂̊̋̔̊͑͛͋̈́͜͜ͅS̸̝͎͓̣̺͔͌̉͊̒̇H̶̲͔̺͚͉̠̤͔̙̝̲̏͒͆̉̇͜

**Comment by Qtrick** [comment]
I don't rly understand this step: "Open chrome vox(ctrl+alt+z) then click resources (Like open the menu which has the links like the Search TO method from the old methods), then click on one of the links."

I activated chrome vox on the no wifi screen but I'm not able to find the resources portion. Some clarification or visual would help.

**Reply by v4pl** [reply]
oh @Qtrick so basiclly after u click ctrl alt z u have to press "search" button on your keyboard and the letters "O" and "T" if u need help join our discord sever https://discord.gg/VQr7E7D7CV

**Comment by Endy3838** [comment]
I can't do the chrome vox thing it got blocked

**Reply by v4pl** [reply]
lol

**Comment by wvminecraftkid** [comment]
I'm on 120 and I can't get art shift s to work

**Reply by AshtonDavies** [reply]
> I'm on 120 and I can't get art shift s to work 

Might wanna double check your version. That part was patched in v121.

**Reply by v4pl** [reply]
ya also u have to do it fast enough

**Reply by Blobby-Boi** [reply]
For me, alt+shift+s doesnt work on secure test browser.
if you are using secure test browser just try another kiosk app.
If you are not, then try do it instantly after you open the app.
if it still doesnt work, try another kiosk app.

**Reply by eaglerstupid** [reply]
yeah, mine doesnt work either im doing it INSTANTLY after i click on my kiosk app and no work :( (im on ver 120)

**Reply by eaglerstupid** [reply]
like half of it is patched

**Reply by v4pl** [reply]
only works on version 120.

**Reply by Gaboom63** [reply]
> I'm on 120 and I can't get art shift s to work

@vaperbest @eaglerstupid @Blobby-Boi @AshtonDavies @wvminecraftkid As some of y'all might know around September maybe October there is a new unenrollment method coming out but I think you need to be bellow v123 or v124 so if you see a new method from v124 or v126 (of skioviox) if you care about the future method for the love of everything do not update. Hope this helps one of yall down the line!

**Reply by AshtonDavies** [reply]
> > I'm on 120 and I can't get art shift s to work
> 
> @vaperbest @eaglerstupid @Blobby-Boi @AshtonDavies @wvminecraftkid As some of y'all might know around September maybe October there is a new unenrollment method coming out but I think you need to be bellow v123 or v124 so if you see a new method from v124 or v126 (of skioviox) if you care about the future method for the love of everything do not update. Hope this helps one of yall down the line!

You could probably just downgrade when it comes around unless it's locked by KV4 or something. I don't pay much attention to that.

**Comment by Gaboom63** [comment]
You know how downgrading bouta work. It won't let you that far if your on
126

**Reply by v4pl** [reply]
Yes depending on your computer I can downgrade all the way to 118.
